Makefile

```diff
@@ -166,8 +166,9 @@ test/addons/.buildstamp: config.gypi \
 	# Cannot use $(wildcard test/addons/*/) here, it's evaluated before
 	# embedded addons have been generated from the documentation.
 	for dirname in test/addons/*/; do \
```

Comment:
I was thinking about silencing this in the other commit and it landed before I got a chance to get back to it. Can we do it in this? I am okay without that as well.